src/blueapi/startup/simmotor.py

```diff
@@ -90,5 +90,5 @@ def __init__(self, *, timeout: float, **kwargs) -> None:
         super().__init__(**kwargs)
         self._timeout = timeout
 
-    def set(self, value: float) -> Status:
-        return Status(timeout=self._timeout)
+    def set(self, value: float) -> MoveStatus:
+        return MoveStatus(positioner=None, target=None, timeout=self._timeout)
```

Comment:
```suggestion
        return MoveStatus(positioner=self, target=float, timeout=self._timeout)
```
We know these values